### DISCUSSION TITLE
Encrypting stored images

### DISCUSSION THREAD

**Discussion by xpwmaosldk** [immich-app/immich#1150] [Q&A] [answered]
I am planning to use immich for my family and friends, but there is one worrying part. 
In my opinion, many people will upload all their images, but the server administrator can see the user's private photos. Is there a way to encrypt the images being stored?

**Comment by alextran1502** [comment, marked as answer]
There is not at the moment. Encryption at rest would significantly reduce performance of Immich and that is not planned

**Reply by bo0tzz** [reply]
Encryption also isn't feasible because the server needs to access image data in order to do the things Immich does.
